components/VcItem.tsx

```diff
@@ -126,6 +131,18 @@ export const VcItem: React.FC<VcItemProps> = (props) => {
   const context = useSelector(service, selectContext);
   const verifiableCredential = useSelector(service, selectVerifiableCredential);
   const emptyWalletBindingId = useSelector(service, selectEmptyWalletBindingId);
+  const settingsService = appService.children.get('settings');
+  const storeError = useSelector(service, selectStoreError);
+  const isSavingFailedInIdle = useSelector(service, selectIsSavingFailedInIdle);
+  const vcLabel = useSelector(settingsService, selectVcLabel);
+  const DISMISS = () => service.send(VcItemEvents.DISMISS());
+
+  let storeErrorTranslationPath = 'errors.savingFailed';
+  const isDiskFullError =
+    storeError?.message?.match('No space left on device') != null;
```

Comment:
is there any different way to identify the error?

Previous we had to use `SQLITE_FULL` as that was the only way async storage provided. I am wondering if mmkv has a better way? (as message can modified any time in newer release which is a risk)

Reply:
This is the message we get when storage is not found in device
`LOG  Message -  write failed: ENOSPC (No space left on device) -> console.log('Message - ', error.message);`

Are you suggesting like replace equality check :  `No space left on device` -> `ENOSPC`?

Reply:
we have used "No space left on device" since it is more readable compared to ENOSPC